libs/pmp-web/repository/reviewer-statistics/feature/src/lib/containers/reviewer-statistics/reviewer-statistics.component.html

```diff
@@ -6,17 +6,26 @@
     [avatarUrl]="userAvatarUrl || (userStatistics$ | async)?.avatarUrl"
     label="Pull requests pending"
   ></pmp-header-container>
-  <div *ngFor="let repo of (userStatistics$ | async)?.repositories">
-    <div class="user-statistics__repository-label">
+  <div class="user-statistics__repositories">
+    <mat-card
+      *ngFor="let repo of (userStatistics$ | async)?.repositories"
+      class="user-statistics__repository-card"
+      matRipple
+      (click)="onSelectRepository(repo)"
+      [ngClass]="{
+        'user-statistics__repository-card--active': selectedRepositories.includes(repo)
```

Comment:
Avoid using method in the template if it's possible selectedRepositories.includes